fibonacci.py

```diff
@@ -0,0 +1,7 @@
+def fibonacci(n):
+    phi = (1 + (5 ** 0.5)) / 2
+    return int((phi**n - (1 - phi)**n) / (5 ** 0.5))

```

Comment:
math mistake
```suggestion
    psi = (1 - (5 ** 0.5)) / 2
    return int((phi**n - psi**n) / (5 ** 0.5))

```
(1 - (5 ** 0.5)) / 2 **!=** 1 - (1 + (5 ** 0.5)) / 2